source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,264 @@
+         :file:`Downloads` directory.
+      #. When the file finishes downloading, select the
+         :guilabel:`Launch Instances` to proceed to the :guilabel:`Launch
+         Status` menu shown in figure 8.
+
+         .. figure:: figures/aws-web-8.png
+            :scale: 50 %
+            :alt: Launch Status
+
+            figure 8: :guilabel:`Launch Status`
+
+      #. Once the :guilabel:`Launch Status` page changes to what is shown in
+         figure 9, select the :guilabel:`View Instances` button to view your
+         :guilabel:`Instances` dashboard.
+
+         .. figure:: figures/aws-web-9.png
+            :scale: 50 %
+            :alt: View Instance
+
+            figure 9: :guilabel:`View Instance`
+
+Connect to your Clear Linux OS Basic instance
+*********************************************
+
+Your :guilabel:`Instances` Dashboard is shown in figure 10 with the new |CL|
+OS basic instance already selected and in the running state. If there are
+other instances available, they will also be listed but not selected.
+   
+
+.. figure:: figures/aws-web-10.png
+   :scale: 50 %
+   :alt: Instance Dashboard
+
+   figure 10: :guilabel:`Instance Dashboard`
+
+To connect to your running instance, click the :guilabel:`Connect` button
+located at the top of your dashboard.  This will bring up the pop-up dialog
+box shown in figure 11 describing how to connect to your running instance. 
+
+.. figure:: figures/aws-web-11.png
+   :scale: 50 %
+   :alt: Connect to Your Instance
+
+   figure 11: :guilabel:`Connect to Your Instance`
+
+#. Open a terminal on your system.  You should be in your :file:`$HOME`
+   directory.
+#. Copy the previously downloaded keyfile you created earlier from the
+   :file:`Downloads` directory to the current directory.
+
+   .. code-block:: console
+
+      cp Downloads/AWSClearTestKey.pem .
+
+#. Change the attributes of the :file:`AWSClearTestKey.pem` using the
+   :command:`chmod` as instructed in the dialog box shown in figure 11.
+
+   .. code-block:: console
+
+      chmod 400 AWSClearTestKey.pem
+
+#. Select the entire text highlighted in the :guilabel:`Example:` that is
+   shown in figure 11 and paste it into your terminal and add the following
+   text to the end of the command string: `-l clear`.  Press the :kbd:`Enter`
+   key to execute the command.
+
+   .. code-block:: console
+
+      ssh -i "AWSClearTestKey.pem" root@ec2-34-209-39-184.us-west-2.compute.amazonaws.com -l clear
+
+#. A message will appear on the terminal the the authenticty of the host can't
+   be established and prompts you with the message:
+
+   .. code-block:: console
+
+      The authenticity of host 'ec2-34-209-39-184.us-west-2.compute.amazonaws.com (34.209.39.184)' can't be established.
+      ECDSA key fingerprint is SHA256:LrziT5Ar66iBTfia8qmiIsrfBUm/UGam76U8bDR6yJc.
+      Are you sure you want to continue connecting (yes/no)?
+
+#. type `yes` and hit the :kbd:`Enter` key.  A warning will be printed to the
```

Comment:
Assuming you agree with the previous comment, we can keep "A warning" un-indented.

Reply:
I changed this to "Another warning is printed ..." and got rid of the will word